docs/examples/shared/example-captions.vtt

```diff
@@ -1,19 +1,19 @@
 WEBVTT
 
-00:00.700 --> 00:04.110
+00:00.700 --> line:0
```

Comment:
this file shouldn't have gotten checked in 😆

Reply:
Yeah, I'm not sure what's going on here...

Reply:
it was something I was playing around with back in the other PR and got copied over

Reply:
I'll remove. I copied all changes over from your PR.